release/device-virtual-go.yaml

```
@@ -0,0 +1,19 @@
+---
+name: 'device-virtual-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: 'ce5f781da4ac4c2a0d7531ae22c293687cd2d279'
```

```suggestion
commitId: 'd6a2d4438fc175be031f77f972aa414c83442272'
```

This is the final commit for the release.